src/ucp/core/ucp_ep.c

```diff
@@ -194,6 +195,8 @@ ucs_status_t ucp_ep_create_base(ucp_worker_h worker, const char *peer_name,
         goto err_free_ep_control_ext;
     }
 
+    ucs_vfs_obj_set_dirty(worker, ucp_worker_vfs_refresh);
```

Comment:
pls add a comment like:
/* Create endpoint representer node, on demand to avoid memory bloat */